SabreTools.Models/PlayStation4/AppPkgHeader.cs

```diff
@@ -0,0 +1,238 @@
+using System.Runtime.InteropServices;
+
+namespace SabreTools.Models.PlayStation4
+{
+    /// <see href="https://www.psdevwiki.com/ps4/PKG_files"/>
+    [StructLayout(LayoutKind.Sequential)]
+    public class AppPkgHeader
+    {
+        /// <summary>
+        /// 0x7F434E54
+        /// ".CNT"
+        /// </summary>
+        [MarshalAs(UnmanagedType.ByValArray, SizeConst = 4)]
+        public byte[]? Magic;

```

Comment:
Please use `uint` here since the size is constrained to 4 bytes and it's not text.

Reply:
I was following https://github.com/SabreTools/SabreTools.Models/blob/main/SabreTools.Models/PlayStation3/SFOHeader.cs

Reply:
In SFO, that should actually be a 4-length ASCII string since it's actually text.

Reply:
The current solution of using `uint` in both places is fine for now.